pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -33,7 +33,9 @@ func TestReconcile(t *testing.T) {
 	logf.SetLogger(logf.ZapLogger(true))
 	username := "johnsmith"
 	userID := uuid.NewV4().String()
-	config := configuration.LoadConfig()
+	cfg, err := configuration.LoadConfig(test.NewFakeClient(t))
+	require.NoError(t, err)
+	config := cfg
```

Comment:
could you please tell me why you reassign the config variable? Wouldn't it be shorter to just assign it directly to `config`?:
```suggestion
	config, err := configuration.LoadConfig(test.NewFakeClient(t))
	require.NoError(t, err)
```

Reply:
yeah, definitely didn't mean to do that, must have been a bit tired when i did this lol, will fix

Reply:
fixed here: a7cfb415bcaa14a4fe90ea746eb7dfc59185eb65